### ISSUE TITLE
java.lang.IllegalStateException: Could not load module definition from: VirtualFile

### ISSUE THREAD

**Issue by Dougie008** [jeremymailen/kotlinter-gradle#216]
hi, when i upgradle gradle plugin version to 7.0.0 and gradle version to 7.0.2, i get the below error.
kotlinter version is `3.4.5`. 
```
java.lang.IllegalStateException: Could not load module definition from: VirtualFile: /Users****s/Home!/modules/java.base/module-info.class. The file might be broken by incorrect post-processing via bytecode tools. Please remove this file from the classpath.
	at org.jetbrains.kotlin.resolve.jvm.modules.JavaModuleInfo$Companion.read(JavaModuleInfo.kt:109)
	at org.jetbrains.kotlin.cli.jvm.modules.CliJavaModuleFinder.findSystemModule(CliJavaModuleFinder.kt:53)
	at org.jetbrains.kotlin.cli.jvm.modules.CliJavaModuleFinder.access$findSystemModule(CliJavaModuleFinder.kt:28)
	at org.jetbrains.kotlin.cli.jvm.modules.CliJavaModuleFinder$systemModules$1.invoke(CliJavaModuleFinder.kt:46)
	at org.jetbrains.kotlin.cli.jvm.modules.CliJavaModuleFinder$systemModules$1.invoke(CliJavaModuleFinder.kt:46)
	at kotlin.sequences.TransformingSequence$iterator$1.next(Sequences.kt:210)
	at kotlin.sequences.FilteringSequence$iterator$1.calcNext(Sequences.kt:170)
	at kotlin.sequences.FilteringSequence$iterator$1.hasNext(Sequences.kt:194)
	at kotlin.sequences.FlatteningSequence$iterator$1.ensureItemIterator(Sequences.kt:316)
	at kotlin.sequences.FlatteningSequence$iterator$1.hasNext(Sequences.kt:303)
	at kotlin.sequences.SequencesKt___SequencesKt.none(_Sequences.kt:1921)
```

**Comment by oheyadam** [comment]
Experiencing this too. Can you share which Kotlin and JDK versions you're using? Are you also using Arctic Fox?

**Comment by oheyadam** [comment]
Running through the same problem too. Can you share which Kotlin version you're using?

**Comment by jeremymailen** [comment]
Thanks for the report, can you share more about your build configuration?
Wasn't able to repo the issue on Gradle 7. Passed through those versions and at gradle 7.1.1 at the moment which runs ok.
I haven't seen an error similar to this before, but looks kinda like: https://youtrack.jetbrains.com/issue/KT-44735
which indicates it might be related to the kotlin version you are using.

**Comment by oheyadam** [comment]
Thanks, maybe it is Kotlin? I'll explore more later. My current config is Kotlin is Kotlin 1.4.20, AGP 4.2.0, and I was trying to update to JDK 11, from 8.

**Comment by jeremymailen** [comment]
Ok, noted. To my knowledge, this should work. In one project I'm using kotlinter-gradle `3.4.5`, kotlin `1.4.32`, java 15, gradle 7.1.1. Have also tried same on a project with Android tools 4.2.0.

**Comment by Dougie008** [comment]
> Running through the same problem too. Can you share which Kotlin version you're using?
hi, i am using kotlin 1.5.10

**Comment by jeremymailen** [comment]
I just released `3.5.0` which bumps min kotlin to 1.5.0. Give this a try and see if it fixes compatibility?

**Comment by Dougie008** [comment]
still same error, but very intersting, in my another project(same environment, i had checked and ensure the config file) work well.
